File: chat/kit_conversation/src/main/java/io/chat/conversation/ConversationRoute.java
package io.chat.conversation;


import android.content.Context;
import android.content.Intent;

import io.chat.conversation.ui.ConversationActivity;
import io.im.uicommon.helper.RouteHelper;

/**
 * by DAD FZ
 * 2026/6/10
 * desc：
 **/
public class ConversationRoute {

    public static void goConversation(Context context) {
        RouteHelper.go(context, new Intent(context, ConversationActivity.class));
    }
}

File: chat/kit_conversation/src/main/java/io/chat/conversation/ConversationRoute.kt
package io.chat.conversation

import android.content.Context
import io.im.uicommon.route.IMRoute
import io.im.uicommon.route.RouterConstant


/**
 * by DAD FZ
 * 2026/6/11
 * desc：
 **/
object ConversationRoute {

    fun goConversation(context: Context) {
        IMRoute.withKey(RouterConstant.PAGE_CONVERSATION_PAGE)
            .withContext(context)
            .navigate()
    }
}